src/main/java/com/microsoft/appcenter/event/EventReporter.java

```diff
@@ -17,20 +17,22 @@
     private Description lastDescription = null;
     private int labelCounter = 0;
     private int screenshotCounter = 0;
+
+    // Parameters will always be in "[]": https://github.com/junit-team/junit4/blob/7111b9621997f6c660687b8ac04003398343ee3a/src/main/java/org/junit/runners/Parameterized.java#L484
     private final Pattern paramsFinder = Pattern.compile("\\[(.+?)\\]");
```

Comment:
I know you did not touch this regex, but it looks suspicious... Firstly I guess '.+?' is equivalent to '.*'? Second it matches on the entire string "[oneThing[onsubThing]] [somethingElse]"